test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -19,6 +19,13 @@
 #define AM_MSG_ID 0
 
 
+const char *tag_recv_nb_name      = "ucp_tag_recv_nb";
+const char *tag_send_nb_name      = "ucp_tag_send_nb";
+const char *am_send_nbx_name      = "ucp_am_send_nbx";
+const char *am_recv_data_nbx_name = "ucp_am_recv_data_nbx";
+const char *conn_id_recv_name     = "conn_id receive";
```

Comment:
why need to add these vars? each of them is used only once

Reply:
to save its address in request's field

Reply:
we can save the pointer to the string literal itself

Reply:
need to treat these as `const char *`